app/models/commit_monitor_branch.rb

```diff
@@ -20,6 +20,11 @@ def self.github_commit_uri(user, repo, sha = "$commit")
     "https://github.com/#{user}/#{repo}/commit/#{sha}"
   end
 
+  def enabled_for?(checker)
```

Comment:
I like this but this should probably call `repo.enabled_for?(checker)` as checkers are enabled/disabled at the repo level.  @Fryguy thoughts?